src/pages/blog/[...slug].astro

```diff
@@ -12,17 +12,48 @@ export async function getStaticPaths() {
 
 const { entry } = Astro.props;
 const { Content } = await entry.render();
+
+// Helper to format date
+const formatDate = (date: Date) => {
+  return date.toLocaleDateString("en-US", {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric'
+  });
+};
```

Comment:
You should do this with some sort of internationalization and stuff across the entire app or dependent on the messages we show